src/net/sourceforge/kolmafia/KoLDesktop.java

```diff
@@ -250,14 +251,9 @@ static final void updateTitle() {
       KoLDesktop.INSTANCE.setTitle(KoLDesktop.INSTANCE.lastTitle);
     }
 
-    Frame[] frames = Frame.getFrames();
-    for (int i = 0; i < frames.length; ++i) {
-      if (frames[i] instanceof GenericFrame) {
-        GenericFrame frame = (GenericFrame) frames[i];
-
-        frame.setTitle(frame.getLastTitle());
-      }
-    }
+    Arrays.stream(Frame.getFrames())
+        .filter(f -> f instanceof GenericFrame)
+        .forEach(f -> f.setTitle(((GenericFrame) f).getLastTitle()));
```

Comment:
In `Kolmafia.java`, you use `GenericFrame.class::isInstance`, etc. Better to be consistent I think.